src/SceneKit/Defs.cs

```diff
@@ -283,7 +283,7 @@ public enum SCNAntialiasingMode : ulong {
 		None,
 		Multisampling2X,
 		Multisampling4X,
-#if MONOMAC
+#if MONOMAC || __MACCATALYST__
```

Comment:
Is catalyst support new new in Xcode 13 ?

In that case there should be a `[MacCatalyst (15,0)]` on each member

Reply:
No, it's not new new! I think I only got this because of the xtro - header files don't mention it, and web docs indicate 13.1 https://developer.apple.com/documentation/scenekit/scnantialiasingmode